src/utils/control-button.js

```diff
@@ -0,0 +1,31 @@
+//@flow
+import {h, Component} from 'preact';
+import style from '../styles/style.scss';
+
+/**
+ * withEventManager component
+ * @param {Component} componentName - the component name
+ * @returns {React$Element} - component element
+ */
+const controlButton: Function = (componentName: string) => (ComponentToWrap: Component) => {
+  return class ControlButtonComponent extends Component {
+    _buttonElement: HTMLElement;
+    /**
+     * render component
+     *
+     * @returns {React$Element} - component element
+     * @memberof ControlButtonComponent
+     */
+    render(): React$Element<any> {
+      return (
+        <div
+          ref={c => (c ? (this._buttonElement = c) : undefined)}
+          className={[style.controlButtonContainer, `${__CSS_MODULE_PREFIX__}-${componentName.toLowerCase()}-control`].join(' ')}>
```

Comment:
I think this might be a bit bigger than what I initially thought.
Need to take into account - sometimes the styles are indeed coming from the styles.scss file so we don't necessirily want to give up on this.
Secondly, this is adding another layer of wrapping around our components, which can easily be done with a small helper function instead.

Original issue is due to fact that adding an empty class gets minimized by the css loader minifier.
https://stackoverflow.com/questions/42823494/retaining-empty-unused-classes-with-css-loader
https://github.com/css-modules/css-modules/issues/140

Reply:
There's no problem with styles comes from `style.scss` (for example like in `Volume` component). The only thing the decorator does is to append the appropriate class name on the top of the control button element. If styles are persist it will apply the settings from the file.

Reply:
but we are adding an additional div here, not the expected behviour.
I would even go to simplyfing it to add hardcoded string values where ever a scss class doesn't exist.